File: pset1/mario/mario.c
#include<cs50.h>
#include<stdio.h>
#include<stdio.h>
void half_pyramid(int n);

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height <= 0 || height > 8);
    half_pyramid(height);
}
void half_pyramid(int n)
{
    int spaces;
    int dashes;
    for (int i = 2; i <= n + 1; i++)
    {
        for (spaces = (n - i); spaces >= 0; spaces--)
        {
            printf(" ");
        }
        for (dashes = 1; dashes < i; dashes++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/mario/less/mario.py
from cs50 import get_int


def pattern(n):

    k = 2*n - 2

    for i in range(0, n):
        for j in range(0, k):
            print(end=" ")
        k = k - 2
        for j in range(0, i+1):
            print("# ", end="")
        print("\r")


while True:
    n = get_int("Enter the height of the tower : ")
    if n > 0 and n < 9:
        break

pattern(n)